src/Web/WebApp/Components/Common/Cards/CreateEntityCard.razor.css

```
@@ -0,0 +1,25 @@
+/* Dashed Border Card Styles */
+.card-overlay {
+    position: absolute;
+    inset: 0;
+}
+
+.hover-mud-border-primary:hover {
+    border-color: var(--mud-palette-primary) !important;
+}
+
+.hover-bg-surface-container:hover {
+     background-color: rgba(var(--mud-palette-surface-rgb), 0.5) !important;
```

Stylelint (17.2.0)

[error] Expected "rgba" to be "rgb" (color-function-alias-notation)

(color-function-alias-notation)